src/views/SessionView/index.vue

```diff
@@ -299,13 +299,17 @@ export default {
     top: 20px;
     left: unset;
     right: 20px;
-    width: 400px;
+    width: 300px;
```

Comment:
The chat was taking up a lot of space from the toolbar around these dimensions
<img width="533" alt="whiteboard" src="https://user-images.githubusercontent.com/35900194/87724094-d4545e00-c788-11ea-8113-52f2ec896b99.png">